docs/Developers/Meta-review.rst

```diff
@@ -0,0 +1,132 @@
+is shown on coala community website.
+
+At the moment, only THUMBS_UP and THUMBS_DOWN are collected and analyzed,
+as they are most commonly used and express strong and clear feelings:
+THUMBS_UP means the review comment is well-written and helpful, while
+THUMBS_DOWN means the review comment is misleading, or even worse, violates
+`coala Community Code of Conduct <https://github.com/coala/cEPs/blob/master/cEP-0006.md>`__.
+
+.. note::
+
+    Only GitHub is being analyzed at the moment. Support for GitLab will
+    be available in the future.
+
+Meta-review Process
+-------------------
+
+The meta-review process for coala is as follows:
+
+1. Reviewers create their reviews.
+
+2. Anyone, especially author of the pull request, can do meta-review (THUMBS_UP,
+   THUMBS_DOWN) via reactions on GitHub.
+
+3. Meta-review information will be collected and analyzed automatically:
+
+   * Meta-reviewers will be given bonus points to encourage such behaviour.
+
+   * Meta-reviewees will be given positive points (on receiving THUMBS_UP) or
+     negative points (on receiving THUMBS_DOWN). Those points are weighted
+     according to score of meta-reviewers. Those who have higher score in
+     the meta-review system will have more impacts on others, so please do
+     meta-reviews carefully.
+
+.. note::
+
+    Due to limitations on GitHub, review summary
+    (`example <https://github.com/coala/coala-bears/pull/2517#pullrequestreview-125039346>`__)
+    is not able to be meta-reviewed. If your review is not related to any
+    particular line, leaving a comment
+    (`example <https://github.com/coala/coala-bears/pull/2517#issuecomment-393678689>`__)
+    is encouraged.
+
+.. caution::
+
+    Don't **edit** or **delete** your review after it has been meta-reviewed.
+    If you do that, you are destroying feedback from the community. This
+    is considered as an improper behaviour and a negative score will be
+    given to you.
+
+Automated Scoring Process
+-------------------------
+
+A complete ranking list can be found on
+`Meta-review score ranking list <https://deploy-preview-143--coala-community.netlify.com/meta-review/>`__.
+Score is based on number of positive (THUMBS_UP) & negative (THUMBS_DOWN)
+reactions one receives, and number of positive & negative reactions one gives
+away. The formula can be found in
+`cEP-0019.md <https://github.com/coala/cEPs/blob/master/cEP-0019.md#ranking-list>`__.
+Calculation details can be found in
+`meta_review/handler.py <https://github.com/li-boxuan/community/blob/2143fbcf925beb1ffe5ab1d9dd4d177e827dece2/meta_review/handler.py>`__.
+
+.. note::
+
+    The scoring process is automated, and will be polished from time to
+    time. As a meta-reviewer, you don't need to care too much about the
+    details of meta-review system. Just make sure you follow the meta-review
+    process!
+
+Iterative Weighting Factor
+--------------------------
+
+As mentioned in the last section, P1 and P2 are weighted. They are updated
```

Comment:
avoid mentioning `P1` and `P2`.

Just use a high level textual description of the algorithm's *intent*.